src/net.cpp

```diff
@@ -2171,7 +2174,7 @@ void CNode::AbortMessage() UNLOCK_FUNCTION(cs_vSend)
     LogPrint("net", "(aborted)\n");
 }
 
-void CNode::EndMessage() UNLOCK_FUNCTION(cs_vSend)
+void CNode::EndMessage(const char* pszCommand) UNLOCK_FUNCTION(cs_vSend)
```

Comment:
I think it's to invasive for a little logging feature. It would basically require to change all `PushMessage(...)` then `BeginMessage()` and `EndMessage()` function. An either conversion of CMessageHeader (that it accepts `std::string`) or some conversions to c-strings in the mentioned functions.
IMO, it's not worth changing this.